backend/src/domain/error.ts

```diff
@@ -0,0 +1,20 @@
+export const DatabaseMigrationError = 'DATABASE_MIGRATION is optional, but if set must be one of [UP, DOWN]';
+export const DatabaseConnectionError = 'Missing environment variables for database connection';
+export const EntryNotFoundError = (tableName: string, filter: any) =>
+  `Unable to find entry in ${tableName} with filter [${JSON.stringify(filter)}]`;
+export const InsertError = (resultLength: number) => `Insert expected one element but was ${resultLength}`;
+export const MealNotFoundError = (mealName: string) => `Couldn't find meal with name:  ${mealName.toUpperCase()})`;

```

Comment:
As per my previous suggestion of creating common re-usable error types this would not be needed.
Instead we'll throw a `DatabaseNotFoundError` and pass in the meal-specific message.